headers/functions/arm7.h

```diff
@@ -1,7 +1,7 @@
 #ifndef HEADERS_FUNCTIONS_ARM7_H_
 #define HEADERS_FUNCTIONS_ARM7_H_
 
-void EntryArm7(void);
+void _start(void);
```

Comment:
Can we name this something else? Like `_start_arm7`? I don't want this to match the arm9 symbol when they aren't copies.

Reply:
In the actual codebase, it is required that this function is named `_start()` in both ARM9 and ARM7. Though I'm fine with renaming the ARM7 function here.